tgui/packages/tgui/components/Popper.tsx

```diff
@@ -43,7 +46,7 @@ export class Popper extends Component<PopperProps> {
       // This code is copied from `findDOMNode` in inferno-extras.
       // Because this component is written in TypeScript, we will know
       // immediately if this internal variable is removed.
```

Comment:
youll want to remove this entire hack comment

Reply:
adn then also make sure this actually works lol idk if this was a fundamental issue with refs or if it was just an infenro bug